script.js

```diff
@@ -1,25 +1,37 @@
 var player = 1;
 var jogadas = 0;
 var ha_vencedor = false;
-var valida_posicao =  new Array(0,0,0,0,0,0,0,0,0);
-var marca_posicao = new Array();
+var valida_posicao = [
+    [false,false,false], // valida_posicao[0][0],valida_posicao[0][1],valida_posicao[0][2],
+    [false,false,false], // valida_posicao[1][0],valida_posicao[1][1],valida_posicao[1][2],
+    [false,false,false], // valida_posicao[2][0],valida_posicao[2][1],valida_posicao[2][2],
+];
 
-function clic(posicao_jogada){
+var marca_posicao =  [
+    [false,false,false],
+    [false,false,false], 
+    [false,false,false], 
+];
+
+document.getElementById("vez").innerHTML = 'Vez de O';
+function clic(coluna,linha){
```

Comment:
Just a small typo.
```click```

Reply:
I changed the function name for `play`, because function with name` click` is giving conflict with other function.